site/content/api/Command-And-Control-API.md

```diff
@@ -16,50 +16,46 @@ The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e
 
 # Operations
 
-The following API defines operations that can be used by *Business Applications* to send control messages to devices over an *AMQP 1.0 Network* and how to get the response.
+The following API defines operations that can be used by *Business Applications* to send commands to devices over an AMQP 1.0 Network and get a response in return.
 
 ## Send a Command
 
 **Preconditions**
 
-1. The *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
-2. *Business Application* has established an AMQP link in role *sender* with the address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send control messages for and `${tenant_id}` is the ID of the tenant the device belongs to. 
+1. *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
+2. *Business Application* has established an AMQP link in role *sender* with the target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send messages to and `${tenant_id}` is the ID of the tenant that the device belongs to.
```

Comment:
The application also needs to open a receiver link on the *reply-to* source address in order to receive the response, doesn't it?